packages/spatial/src/transform/components/EntityTree.test.tsx

```diff
@@ -639,6 +650,183 @@ describe('useChildWithComponent', () => {
   })
 })
 
+describe('useChildrenWithComponent', () => {
+  // Run before every test case
+  beforeEach(() => {
+    createEngine()
+  })
+  afterEach(() => {
+    return destroyEngine()
+  })
+
+  it('returns the closest children entities that has the requested components', async () => {
+    // Initialize with dummy data for the test
+    let rootEntity = createEntity()
+    let child_1 = createEntity()
+    let child_2 = createEntity()
+    let results = [UndefinedEntity]
+    const component = HighlightComponent
+    const component2 = VisibleComponent
+
+    // Define the Reactor that will run the tested hook
+    const Reactor = () => {
+      //TODO pick up here where i left off in tests, need to properly convert result/entity to an array
+      //DO NOT COMMIT MEDO NOT COMMIT MEDO NOT COMMIT MEDO NOT COMMIT MEDO NOT COMMIT MEDO NOT COMMIT MEDO NOT COMMIT
```

Comment:
Hey @SamMazerIR
This was actually commited to dev, including the warning.
What is this warning about? What is missing in these tests?